include/libtorrent/aux_/proxy_settings.hpp

```diff
@@ -79,6 +79,10 @@ namespace aux {
 		// This is only supported by SOCKS5 and HTTP.
 		bool proxy_hostnames = true;
 
+		// defaults to false for integration compatibility. It means that the
+		// torrent port is listening with tcp and udp while there is a proxy configured.
+		bool listen_on_proxy = false;
```

Comment:
This name seem to suggest we're listening for connections from the *proxy*. I have a feeling that `accept_incoming` would be a better name for this.

Reply:
Thx! fixed in [91081ce](https://github.com/arvidn/libtorrent/pull/7726/commits/91081ce7eb5eff128df3f8fe1f8294f95c1675aa)